src/locales/locales.json

```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
```

Comment:
looks like a conflict was merged

Reply:
Thanks for spotting, didn't "Command-S" enough in my open windows 😔